### PR TITLE
Card

### PR DESCRIPTION
**Pull request by Tanay-J**
card.css
- vertical and horizontal card
- card with detailed description
- card with a badge
- card with text overlay
- dismissable card
- text only card

image
image
image
image

### CODE REVIEW

File: components/card.css [hunk 1]
```diff
@@ -0,0 +1,67 @@
+
+.card-vertical {
+    display: flex;
+    flex-direction: column;
+    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
```
Comment (AnuragSahu11) [inline components/card.css]: You could create a shadow class , just a suggestion

**Review by AnuragSahu11 — APPROVED** [review]
Looks Good

File: components/card.css [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+}
+.card-details{
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    overflow: hidden;
+}
+.sub-title{
+    color: var(--gray-dark);
+    font-size: 0.75rem;
```
Comment (Hamdrive) [inline components/card.css:30]: Try to stick to pixels for small values

File: components/card.css [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+    display: none;
+}
+.grid-row-span-2{
+    grid-row: span 2;
+}
+.position-rel{
+    position: relative;
+}
+
+
```
Comment (Hamdrive) [inline components/card.css:68]: take care of excess spacing, use good formatting practices

**Review by Hamdrive — COMMENTED** [review]
Good work done, changes suggested

File: components/card.css (hunk 1 above)
Comment (Tanay-J) [reply components/card.css]: Good point. I've changed it.

File: components/card.css (hunk 2 above)
Comment (Tanay-J) [reply components/card.css:30]: I wanted this ```font-size``` for ```sub-title``` class to be linked to root font size, which in this case is 16px.

**Pull request merged**